policy.md

```diff
@@ -1,133 +1,232 @@
+
+Exceptions to providing these audit statement URLs:
 
 * The SHA-256 fingerprint of the certificate is specifically listed as in scope in the audit statements of the parent certificate, and the "Audits Same as Parent" checkbox is checked; or
-* The certificate is issued and managed under the same CP, CPS, or CP/CPS as the parent certificate and the "CP Same as Parent", "CPS Same as Parent", and/or "CP/CPS Same as Parent" checkbox is checked; or
 * The certificate has expired; or
-* The certificate has been revoked, and the corresponding record in the CCADB has been updated with the correct revocation status.
+* The certificate has been revoked, and the corresponding record in the CCADB has been updated with the correct revocation status; or
+* The certificate was issued after the audit period of the most recently performed audit. In such cases, the certificate MUST be included in the scope of the CA Owner's next periodic audit.
+
+CA Owners MUST:
+- submit an [Add/Update Root Request Case](https://www.ccadb.org/cas/updates) to add or update audit statements for root CA certificates stored in the CCADB.
+- add or update audit statements for subordinate CA certificates directly on the record in the CCADB (unless the exceptions stated above apply).
+
+If an audit or audit statement disclosed to the CCADB does not meet the requirements of this policy, then a Root Store Operator MAY require that the CA Owner obtain a new audit, at the CA Owner's expense, for the period of time in question. Additionally, depending on the nature of concerns with the audit, a Root Store Operator MAY require that the CA Owner obtain such an audit from a new auditor.
 
-CA Owners MUST submit an [Add/Update Root Request Case](https://www.ccadb.org/cas/updates) to add or update these documents for root CA certificates stored in the CCADB.
+The presence of qualifications in an audit statement is not, by itself, generally considered a reason to remove a CA Owner from a Root Store. The purpose of audits is to honestly and thoroughly assess a CA Owner’s compliance with requirements which are necessary to assure a secure and stable ecosystem. Audit findings, including qualifications, can help to identify opportunities for improvement, whether for individual CAR Owners or for the wider industry as a whole.
 
-CA Owners MUST add or update these documents for subordinate CA certificates directly on the record in CCADB (unless the exceptions stated above apply).
+### 5.1 Scope of Certificates Covered by an Audit
 
-### 5.1 Audit Statement Content ###
+CA certificates MUST appear in the Issuer’s annual audit statement, beginning with the statement corresponding to the audit period when the certificate was issued. Those certificates MUST continue to appear in audit statements until having appeared on at least one audit statement following revocation or key destruction (observed and reported by a Qualified Auditor).
```

Comment:
This implies that revocation is sufficient to stop auditing a subordinate CA, but expiration is not. Is that the intent?

Reply:
The implication is unintentional.

```suggestion
CA certificates MUST appear in the Issuer’s annual audit statement, beginning with the statement corresponding to the audit period when the certificate was issued. Those certificates MUST continue to appear in audit statements until having appeared on at least one audit statement following revocation, expiration, or key destruction (observed and reported by a Qualified Auditor).
```